bin/prepare

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require 'date'
+
+begin
+  if ARGV[0]
+    release_date = Date.parse ARGV[0]
+  else
+    release_date = Date.today
+    release_date = release_date.next until release_date.thursday?
+  end
+
+  release_date = release_date.strftime('%Y%m%d')
+
+  `git checkout -b deploy-#{release_date}`
```

Comment:
This assumes that the user is on latest branch. Why can't we set it to fetch latest branch code from GitHub prior to generating new branch?

What will happen if this is ran twice?